yt/frontends/ramses/data_structures.py

```diff
@@ -197,7 +204,7 @@ def __init__(self, ds, domain_id):
         for t in ["grav", "amr"]:
             setattr(self, f"{t}_fn", basename % t)
         self._part_file_descriptor = part_file_descriptor
-        self._read_amr_header()
+        self.max_level = self.ds.parameters["levelmax"] - self.ds.parameters["levelmin"]
```

Comment:
Good catch.  I feel like I've seen this as an issue elsewhere.  Are we correctly setting `min_level` as well?

Reply:
I have to say I'm a bit confused about `max_level`/`min_level`. In RAMSES standard, these are simply related to the number of levels of refinement from the root, i.e. `1/2**level` is the current cell size.
But for yt, if `max_level` is how more refined the deepest level is compared to the base grid, what does `min_level` min? Should it be 0 if there are some cells that are at the resolution of the base grid?